utils/Room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class Room{
+    #private = 123
```

Comment:
?why is this here

Reply:
i was experimenting with private class fields
its been removed